### PR TITLE
GitHub actions latest

### PR DESCRIPTION
**Pull request by Ira-Maier**
just merge:
- GitHub CI files in .github/workflows + config.sh
- scripts/uninstall_development.sh
- status badges for GitHub CI in README --> only shows status of development-5

docu here: https://www.chemotion.net/chemotionsaurus/docs/eln/development/testing_code

passing tests here: https://github.com/ComPlat/chemotion_ELN/actions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,10 @@ see online [docs](https://www.chemotion.net/chemotionsaurus/docs/eln/about)
 
 ## Code Status
 
+[![JavaScript Unit Tests](https://github.com/ComPlat/chemotion_ELN/actions/workflows/testjs.yml/badge.svg?branch=development-5)](https://github.com/ComPlat/chemotion_ELN/actions/workflows/testjs.yml/badge.svg?branch=development-5)
+[![Ruby Unit Tests](https://github.com/ComPlat/chemotion_ELN/actions/workflows/testrb.yml/badge.svg?branch=development-5)](https://github.com/ComPlat/chemotion_ELN/actions/workflows/testrb.yml/badge.svg?branch=development-5)
+[![Acceptance Tests](https://github.com/ComPlat/chemotion_ELN/actions/workflows/testacceptance.yml/badge.svg?branch=development-5)](https://github.com/ComPlat/chemotion_ELN/actions/workflows/testacceptance.yml/badge.svg?branch=development-5)
+
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1054134.svg)](https://doi.org/10.5281/zenodo.1054134)
 
 [![Build Status](https://travis-ci.org/ComPlat/chemotion_ELN.svg?branch=master)](https://travis-ci.org/ComPlat/chemotion_ELN)
```
Comment (PiTrem) [inline README.md:47]: those two lines could be removed for now

File: .github/workflows/testacceptance.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        run: |
+            /bin/bash -l -c "source /home/gitlab-runner/.asdf/asdf.sh && bundle install"
+            /bin/bash -l -c "source /home/gitlab-runner/.asdf/asdf.sh && yarn install"
+
+      - name: precompile
+        working-directory: /home/gitlab-runner/chemotion_ELN
+        run: |
+            /bin/bash -l -c "source /home/gitlab-runner/.asdf/asdf.sh && RAILS_ENV=test bundle exec rake db:migrate db:test:prepare > /dev/null"
+
+            mv public/welcome-message-sample.md public/welcome-message.md
```
Comment (PiTrem) [inline .github/workflows/testacceptance.yml:67]: should not be needed anymore with fff9c3624

**Review by PiTrem — CHANGES_REQUESTED** [review]
minor changes

**Review by Ira-Maier — COMMENTED** [review]
update from pi_r

**Review by Ira-Maier — COMMENTED** [review]
update 2

**Review by Ira-Maier — COMMENTED** [review]
viewed

**Pull request merged**